tamarin-prover/tamarin-prover
# Fix up Travis compilation

This makes a number of minor improvements to the Travis build script:

 - move to container-based infrastructure, enabling caching. This cuts build time to ~8m from 30m+
 - move to the latest way of setting up OCaml
 - run `tamarin-prover test` and check it works
 - add a slack integration for infosec-oxford.slack.com
